services/transformer/features_impl.go

```diff
@@ -11,16 +11,16 @@ import (
 	"github.com/samber/lo"
 	"github.com/tidwall/gjson"
 
-	"github.com/rudderlabs/rudder-go-kit/config"
 	"github.com/rudderlabs/rudder-go-kit/logger"
 	"github.com/rudderlabs/rudder-server/utils/httputil"
 )
 
 type featuresService struct {
 	logger   logger.Logger
 	waitChan chan struct{}
-	config   FeaturesServiceConfig
+	options  FeaturesServiceConfig
```

Comment:
Should we rename this struct as well?
```suggestion
	options  FeaturesServiceOptions
```

Reply:
Done